### DISCUSSION TITLE
Where to download the hssd train datasets?

### DISCUSSION THREAD

**Discussion by Sawyer-Wu** [facebookresearch/habitat-lab#1914] [Q&A]
I loss data and the path should be data/datasets/hssd/rearrange/train/social_rearrange.json.gz. Where can I get this dataset?

**Comment by linghei** [comment]
Excuse me, have you been resolve it ?

**Reply by Sawyer-Wu** [reply]
python -m habitat_sim.utils.datasets_download --uids hssd-hab hab3-episodes habitat_humanoids hab3_bench_assets
